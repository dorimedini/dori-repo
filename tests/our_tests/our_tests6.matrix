{
matrix m5(3,4) = [4,6,17,3;22,5,103,-17;-10,10/2,18/(-3),18/-3];
matrix m6(4,2) = [8,2;4,6;-23,1;0,0];
matrix m7(3,2) = m5*m6;
print m7;
}