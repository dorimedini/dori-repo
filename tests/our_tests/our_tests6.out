[-335,61;-2173,177;78,4]
